shared/constants/network.ts

```diff
@@ -522,64 +506,6 @@ export const IPFS_DEFAULT_GATEWAY_URL = 'dweb.link';
 
 // The first item in transakCurrencies must be the
 // default crypto currency for the network
```

Comment:
I must delete these.

Reply:
deleted in [1a63b0e](https://github.com/MetaMask/metamask-extension/pull/18862/commits/1a63b0e9a7fb003bad55ba7dadb5c03f7156e6b0)